networks/ghostnet/faucet_values.yaml

```diff
@@ -24,15 +24,28 @@ enableCaptcha: true
 # Faucet won't dispense to an address if its balance exceeds this.
 maxBalance: 10000
 
-profiles:
-  user:
-    amount: 100
-  baker:
-    amount: 2001
+# The minimum Tez allowed per request.
+minTez: 1
+# The maximum Tez allowed per request.
+maxTez: 6001
 
 disableChallenges: false
+# Minimum number of challenges required for the minimum Tez request.
+minChallenges: 10
```

Comment:
I made requesting even the minTez take a drop of time. Should take around 10s via nodejs/cli and about 45 in browser

Reply:
do you mean, 4 seconds in browser? CLI should be longer than browser